src/test_SlackNotifier.py

```diff
@@ -6,7 +6,7 @@
 
 class TestSlackNotifier(unittest.TestCase):
     @responses.activate
-    def test_sendSlackNotification(self):
+    def test_send_slack_notification(self):
```

Comment:
Let's revert this change. Since this is a method to test the send_slack_notification method, it'd be more clear to leave it as is.